index/index.go

```diff
@@ -474,6 +474,13 @@ func (idx *Indexer) ensureIndex(
 		fmt.Sprintf("%s-%s", indexTypeRoot, config.Language),
 	}
 
+	// Add a custom normaliser for sort fields.
+	config.Settings.Settings.Analysis.SetNormalizer(
+		"lowercase_trim", OpensearchNormaliser{
+			Type:   "custom",
+			Filter: []string{"lowercase", "trim"},
```

Comment:
For true multilingual support I wonder if this is enough. We should problable look into using something like `icu_collation_keyword` (https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-icu-collation-keyword-field.html) and use the language settings for the index.

Reply:
This would replace the `type` of the sort field and ditch the need for a custom normalizer.

Reply:
Nice! I wonder how that works when sorting across multiple indexes with different collations.

Reply:
Sorting works fine across indexes with different collations. It doesn't return a 400/error and the sort representation seems to be close enough to not result in really strange ordering.